packages/sources/openexchangerates-test/src/endpoint/forex.ts

```diff
@@ -0,0 +1,114 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { PriceEndpoint, PriceEndpointParams } from '@chainlink/external-adapter-framework/adapter'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { customSettings } from '../config'
+
+interface ResponseSchema {
+  disclaimer: string
+  license: string
+  timestamp: number
+  base: string
+  rates: {
+    [key: string]: number
+  }
+}
+
+export type ForexEndpointTypes = {
+  Request: {
+    Params: PriceEndpointParams
+  }
+  Response: SingleNumberResultResponse
+  CustomSettings: typeof customSettings
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema
+  }
+}
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin'],
+    required: true,
+    type: 'string',
+    description: 'The symbol of symbols of the currency to query',
+  },
+  quote: {
+    aliases: ['to', 'market'],
+    required: true,
+    type: 'string',
+    description: 'The symbol of the currency to convert to',
+  },
+} as const
```

Comment:
After the typescript upgrade and the fw bump past 0.25, we want to move to using the satisfies operator here like
```suggestion
} satisfies InputParameters & PriceEndpointInputParameters
```